docs/src/main/sphinx/release/release-440.md

```diff
@@ -0,0 +1,70 @@
+# Release 440 (7 Mar 2024)
+
+## General
+
+* Add support for sub-queries inside `UNNEST` clauses. ({issue}`17953`)
+* Improve performance of {func}`arrays_overlap`. ({issue}`20900`)
+* Change the default value for the `jmxExport` resource group property to
```

Comment:
"Export JMX stats for resource groups by default. This can be turned off via the xxx property"